controllers/users.js

```diff
@@ -410,6 +410,31 @@ const updateSelf = async (req, res) => {
       }
     }
 
+    if (req.body.disabled_roles) {
+      let rolesToDisable;
+      const data = req.body.disabled_roles;
```

Comment:
what if req is null, empty or undefined

Reply:
req object won't be empty or null, if that the case validators will catch that